### PR TITLE
Bugfix: Include ts pref & suffixes in module generation

### PR DESCRIPTION
**Pull request by OlegGulevskyy**
[Original PR](https://github.com/wailsapp/wails/pull/2192) got closed while I was doing poor attempts to resolve conflicts of fork.. :\

### CODE REVIEW

**Review by Goncalo-Marques — COMMENTED** [review]
@OlegGulevskyy @leaanthony Should we mention https://github.com/wailsapp/wails/issues/2058 on this PR description?

File: v2/internal/typescriptify/typescriptify.go [hunk 1]
```diff
@@ -841,11 +841,10 @@ func (t *typeScriptClassBuilder) AddStructField(fieldName string, field reflect.
 	strippedFieldName := strings.ReplaceAll(fieldName, "?", "")
 	fqname := field.Type.Name()
 	classname := "null"
-
 	namespace := strings.Split(field.Type.String(), ".")[0]
-
+	fqname = t.prefix + field.Type.Name() + t.suffix
```
Comment (Goncalo-Marques) [inline v2/internal/typescriptify/typescriptify.go:845]: I think that line 842 is unnecessary, right now:
```suggestion
	fqname := t.prefix + field.Type.Name() + t.suffix
```

**Comment by Goncalo-Marques** [comment]
@OlegGulevskyy, just a health check 🙂 
Any update on this fix?

**Comment by leaanthony** [comment]
@Goncalo-Marques have you tested this pr?

**Comment by leaanthony** [comment]
@OlegGulevskyy There's a small merge conflict here related to #1821. You ok to fix that? I can look at it if you can't?

File: v2/internal/typescriptify/typescriptify.go (hunk 1 above)
Comment (Goncalo-Marques) [reply v2/internal/typescriptify/typescriptify.go:845]: Exactly 🙂

**Comment by leaanthony** [comment]
Thanks @Goncalo-Marques. If @OlegGulevskyy doesn't respond in the next few days I'll resolve the conflict and merge 👍

**Comment by Goncalo-Marques** [comment]
> Thanks Goncalo-Marques. If OlegGulevskyy doesn't respond in the next few days I'll resolve the conflict and merge 👍

Yup, that is probably the best way 😅

**Comment by OlegGulevskyy** [comment]
Hi guys, sorry for the delay, 3rd time having a cold since autumn, weak as a chicken :D 
Gonna resolve the conflicts and we can merge

**Comment by OlegGulevskyy** [comment]
Resolved, looks good on my side

**Comment by leaanthony** [comment]
@Goncalo-Marques can you please confirm this works for you and we'll merge 👍

**Comment by Goncalo-Marques** [comment]
@OlegGulevskyy I think that you didn't address this comment: https://github.com/wailsapp/wails/pull/2227#discussion_r1055727673

**Comment by Goncalo-Marques** [comment]
@leaanthony @OlegGulevskyy Apart from my last comment, everything seems to work properly 😄

File: v2/internal/typescriptify/typescriptify.go (hunk 1 above)
Comment (OlegGulevskyy) [reply v2/internal/typescriptify/typescriptify.go]: indeed, you are right @Goncalo-Marques , it is redundant, removed it

**Pull request merged**